llvm/unittests/CodeGen/GlobalISel/LegalizerHelperTest.cpp

```diff
@@ -3431,6 +3431,45 @@ TEST_F(AArch64GISelMITest, LowerUDIVREM) {
   EXPECT_TRUE(CheckMachineFunction(*MF, CheckStr)) << *MF;
 }
 
+// Test G_SELECT lowering.
```

Comment:
It might be good to add a comment to say we don't expect to legalize scalar selects on aarch64 like this, but it is useful for testing.

Reply:
Good point, added a comment.